r2i/iengine.h

```diff
@@ -62,6 +64,32 @@ class IEngine {
       in_frame,
       r2i::RuntimeError &error) = 0;
 
+  /**
+   * \brief Set an instance of IPreprocessing into Engine.
+   * \param preprocessing Shared pointer on an IPreprocessing object.
+   * \return RuntimeError with a description of an error.
+   */
+  virtual RuntimeError SetPreprocessing (std::shared_ptr<IPreprocessing> preprocessing) = 0;
+
+  /**
+   * \brief Get an instance of IPreprocessing from Engine.
```

Comment:
`Get the IPreprocessing currently configured in the Engine`

Reply:
Solved [here](https://github.com/RidgeRun/r2inference/commit/ff1cbc53dd2061f81b66a1ae50dd41fcc215a065)